### PR TITLE
Fix rwc in including files into compilation

### PR DESCRIPTION
**Pull request by yuit**

### CODE REVIEW

**Comment by mhegazy** [comment]
so what does this do?

File: src/harness/loggedIO.ts [hunk 1]
```diff
@@ -223,7 +223,18 @@ namespace Playback {
                 recordLog.directoriesRead.push(logEntry);
                 return result;
             },
-            (path, extension, exclude) => findResultByPath(wrapper, replayLog.directoriesRead.filter(d => d.extension === extension && ts.arrayIsEqualTo(d.exclude, exclude)), path));
+            (path, extension, exclude) => findResultByPath(wrapper,
+                    replayLog.directoriesRead.filter(
+                        (d) => {
+                            if (d.extension === extension) {
```
Comment (mhegazy) [inline src/harness/loggedIO.ts]: we have multiple reads to the same extension but with different excludes in the same session?

**Comment by mhegazy** [comment]
:+1:

File: src/harness/loggedIO.ts [hunk 2]
```diff
@@ -223,7 +223,18 @@ namespace Playback {
                 recordLog.directoriesRead.push(logEntry);
                 return result;
             },
-            (path, extension, exclude) => findResultByPath(wrapper, replayLog.directoriesRead.filter(d => d.extension === extension && ts.arrayIsEqualTo(d.exclude, exclude)), path));
+            (path, extension, exclude) => findResultByPath(wrapper,
+                    replayLog.directoriesRead.filter(
+                        (d) => {
```
Comment (sandersn) [inline src/harness/loggedIO.ts]: can you remove the parentheses around d?

**Comment by sandersn** [comment]
:+1:

**Pull request merged**